applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_c_phi_reduction_process.cpp

```diff
@@ -0,0 +1,105 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
```

Comment:
I think this was copied from a different test (at least in the git blame I only see Wijtze Pieters name, could you correct this?

Reply:
It seems you are right. Changed on WPK